Makefile

```diff
@@ -289,11 +289,11 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0)
+	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.16.3)
```

Comment:
`controller-gen` depends on kube 1.31 (https://github.com/kubernetes-sigs/controller-tools/blob/v0.16.3/go.mod), which is still OK as we are stuck on 1.29.